env/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "aws_key_path" {}
+variable "aws_key_name" {}
+
+variable "prefix" {
+    description = "Prefix all named resources"
+    default = "<env name>"
+}
+
+variable "aws_region" {
+    description = "EC2 Region for the VPC"
+    default = "eu-west-2"
+}
+
+variable "amis" {
+    description = "AMIs by region"
+    default = {
+        eu-west-2 = "ami-6b3fd60c" # Ubuntu Server 16.04 LTS (HVM), SSD Volume Type
```

Comment:
I'd go for 18.04 directly. It's an LTS too

Reply:
I didn't think 18.04 was released yet. I'll have a look for an AMI and switch it.

Reply:
I think if we don't need any EC2 resources we can probably bin the whole thing.

Reply:
Unless they change something, Canonical has been releasing the LTS version every two years, in April. Next version should be released in April 2020.